benchmarks/Csharp/.editorconfig

```diff
@@ -0,0 +1,223 @@
+# Remove the line below if you want to inherit .editorconfig settings from higher directories
```

Comment:
Is this strictly needed?

Reply:
It freezes linter settings but ... not stricly needed, no.

Reply:
Ah, ok, which linter are you using, and would it be possible to add it to the GitHub Actions setup?
If it's for a linter, having it would be good, yes.
I was just wondering what it was since the file name is `.editorconfig` instead of check style, or lintrc or similar.

Thanks!

Reply:
I was using the Visual Studio 2022 integrated hints, like these:
![image](https://user-images.githubusercontent.com/5293502/160369817-8ca13eb3-a7f8-41dd-a74c-a695cd851d41.png)
For better linting (especially reformatting) we are using Resharper at my main job. The cli tool is free of charge and can be integrated into github actions. (and it can load the editorconfig settings)

Reply:
Thanks, if Resharper is an option, would be great to have similar to CheckStyle, PyLint, RuboCop and the other linter setups :)